bin/aks-create-cluster.sh

```diff
@@ -35,20 +37,26 @@ case "${choice}" in
    * ) echo "Invalid input, Bye!"; exit 1;;
 esac
 
-
 # Who created this cluster.
 CREATOR="${USER:-unknown}"
 # Labels can not contain dots that may be present in the user.name
 CREATOR=$(echo $CREATOR | sed 's/\./_/' | tr "[:upper:]" "[:lower:]")
 
-# Check first to see if SPN exists
-az ad sp show --id ${AKS_SERVICE_PRINCIPAL} > /dev/null 2>&1
-if [ $? -ne 0 ]; then
-    echo "Service Principal \"${AKS_SERVICE_PRINCIPAL}\" not found.  Please create first"
-    exit 1
+# Check first to see if Service Principal has been provide and whether it exist
```

Comment:
s/to see if Service Principal has been provide/to see if a Service Principal has been provided/

Reply:
done